WalletLibrary/WalletLibrary/WalletLibraryVersion.swift

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+/**
+ * The Current Wallet Library Version.
+ */
+struct WalletLibraryVersion {
+    static let High = 0
+    static let Low = 0
+    static let Patch = "1-beta.2"
+    
+    static let Version = "\(High).\(Low).\(Patch)"
```

Comment:
Don't we to distinguish ios from android as well in case formatting only affects one platform but not the other?

Reply:
lol yes that is how I spec'd it, nice catch haha

Reply:
Updated the String in the Network Operation.